manual.adoc

```diff
@@ -149,12 +151,23 @@ Make sure that `~/.local/bin` is listed in the `$PATH` variable and use the appr
 
 You don't have to use `~/.local/bin`, any other path like `~/.cargo/bin` or `/usr/local/bin` will work just as well.
 
+On Windows:
+
+. Download a suitable Windows specific release e.g. `rust-analyzer-x86_64-pc-windows-msvc.zip`
+. Extract the files
+. Copy the `rust-analyzer.exe` file to a suitable directory (e.g. `C:\tools`)
+. Add that directory to the user's *path* _environmental variable_
+. Open a new `Command Prompt` or `PowerShell` to use `rust-analyzer` 
+
 Alternatively, you can install it from source using the command below.
 You'll need the latest stable version of the Rust toolchain.
 
 [source,bash]
 ----
 $ git clone https://github.com/rust-lang/rust-analyzer.git && cd rust-analyzer
+$ git checkout release
+$ cargo build --release
```

Comment:
No, you don't need to do this.

Reply:
By default, it's using the nightly install, hence the checkout release. `cargo xtask` doesn't run without a `cargo build` first, on my system

Reply:
It's working now but there have been a lot of state changes on my end, including upgrading `rustup` and fiddling with rust-analyzer's config. Before it was consistently refusing to recognise the "xtask" part of the cargo command, hence my fallback to `cargo build` first. That was a frustrating waste of time.

I'm not going to do any further diagnosis it works on my system now with the latest rust, rustup and rust-analyzer.